fcrepo-export/src/test/java/org/fcrepo/exporter/ExporterTest.java

```diff
@@ -146,6 +152,18 @@ public void testExportContainer() throws Exception {
         Assert.assertTrue(exporter.wroteFile(new File("target/rdf/rest/1.jsonld")));
     }
 
+    @Rule
+    public ExpectedException expectedEx = ExpectedException.none();
```

Comment:
I can do that and agree it's better... I felt a little weird just doing it just for the purpose of testing.  There's lots of runtime exceptions thrown by the Importer and Exporter and I think a comprehensive look at them is warranted (which ones should halt operation, which ones can be logged without halting, whether the the messages should be written to log files or to standard out)  I thought the following ticket might address that, but it seems to be more limited: https://jira.duraspace.org/browse/FCREPO-2148